spec/rails_helper.rb

```diff
@@ -7,6 +7,7 @@
 require 'rspec/rails'
 
 # Add additional requires below this line. Rails is not loaded until this point!
+require 'capybara/rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.